packages/next/client/image.tsx

```diff
@@ -360,6 +360,14 @@ export default function Image({
         ).join(',')}.`
       )
     }
+    if (
+      layout !== 'fill' &&
+      (Number.isNaN(widthInt) || Number.isNaN(heightInt))
```

Comment:
```suggestion
      (isNaN(widthInt) || isNaN(heightInt))
```

Seems like Number.isNaN() isn't supported in IE.